Website/Pages/ForumIndex/ForumIndex.razor.css

```diff
@@ -1,14 +1,31 @@
+.background {
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+  padding: 2rem;
+  background-image: url(https://imgs.search.brave.com/iBN28HERkONkBG9YeVWhsk6Ju8nnYkUFScUVuUuqacA/rs:fit:1200:1200:1/g:ce/aHR0cHM6Ly9pbWFn/ZXM2LmFscGhhY29k/ZXJzLmNvbS8zNzYv/Mzc2MzM4LnBuZw);
```

Comment:
this image should really be saved as a static asset in `Website/wwwroot/assets`, instead of reaching out to a foreign domain.  is it licensed freely?

Reply:
nothing wrong with using url. and yes random picture from google. well I dont think its matter that much but i will save it to local.